scalr/resource_scalr_workspace.go

```diff
@@ -177,6 +177,11 @@ func resourceScalrWorkspace() *schema.Resource {
 							Optional: true,
 							Default:  true,
 						},
+						"ingress_submodules": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
```

Comment:
Should `policy_group` and `module` resources have the default value for `ingress_submodules` as well?

Reply:
I think yes, updated.